### PR TITLE
Clean up of public docstrings

### PR DESCRIPTION
**Pull request by krisgesling**
#### Description
This is a quick clean up of public docstrings in the primary parse.py and format.py files.

- uses a consistent description for the `lang` parameter
- adds some missing arguments
- add type hints for `fuzzy_match` where the input args are self-explanatory
- autopep8 formatting

#### Type of PR
- [ ] Bugfix
- [ ] Feature implementation
- [ ] Refactor of code (without functional changes)
- [x] Documentation improvements
- [ ] Test improvements

#### Testing
Easiest is eyeballs here I think.

### CODE REVIEW

File: lingua_franca/format.py [hunk 1]
```diff
@@ -290,6 +293,9 @@ def pronounce_number(number, lang='', places=2, short_scale=True,
 
     Args:
         number: the number to pronounce
+        lang (str, optional): an optional BCP-47 language code, if omitted
+                              the default language will be used.
+        places (int): number of decimal places to express number to, default 2
```
Comment (ChanceNCounter) [inline lingua_franca/format.py:298]: "number of decimal places to express" might be better. idk, i can't come up with good phrasing, except that the trailing 'to' is giving me fits

**Comment by ChanceNCounter** [comment]
First pass looks good, marked unread for second pass after lunch.

Props and thanks for those type hints.

**Comment by ChanceNCounter** [comment]
Well, I don't see any code changes (apart from the type hints) and neither does pytest, so, ✅

**Review by ChanceNCounter — APPROVED** [review]

File: lingua_franca/format.py (hunk 1 above)
Comment (krisgesling) [reply lingua_franca/format.py]: Good suggestion - fixed.

**Pull request merged**